config/initializers/sidekiq.rb

```diff
@@ -37,3 +37,29 @@
   config.reaper_timeout  = 150
   config.lock_ttl        = 50.days.to_i
 end
+
+begin
+  # Start the self-destruct scheduler if needed
+  value = ENV.fetch('SELF_DESTRUCT', nil)
+  if value.present? && Rails.application.message_verifier('self-destruct').verify(value) == ENV['LOCAL_DOMAIN']
```

Comment:
Is there a way to re-use the helper here, or have a shared location used here and in the helper?